simtbx/nanoBragg/__init__.py

```diff
@@ -150,7 +151,7 @@ def detector(self):
     det_descr = {'panels':
                    [{'fast_axis': fdet,
                      'slow_axis': sdet,
-                     'gain': self.quantum_gain,
+                     'gain': 1, # this should always be 1 for simulations. If you ran add_noise then quantum gain was multiplied
```

Comment:
This might have been fixed by https://github.com/cctbx/dxtbx/pull/660, but in this case I think its better to leave format class gain = 1 for simulations, and then use the index/integrate phil settings to control gain when e.g. stills_process